tf_encrypted/keras/models/sequential.py

```diff
@@ -106,6 +110,33 @@ def layers(self):
       return layers[1:]
     return layers[:]
 
+  def backward(self, d_y):
+    for layer in reversed(self.layers):
+      grad_weights, d_y = layer.backward(d_y)
+      self._optimizer.apply_gradients(layer.weights, grad_weights)
+
+  def compile(self, optimizer, loss=None):
+    self._optimizer = optimizers.get(optimizer)
+    self._loss = loss
+
+  def fit_batch(self, x, y):
+    y_pred = self.call(x)
+    dy = self._loss.grad(y, y_pred)
```

Comment:
what happens if you compiled with `loss=None`?

Reply:
I think it will crash :)